ui/components/prsidebar/approve.go

```diff
@@ -0,0 +1,54 @@
+package prsidebar
+
+import (
+	"fmt"
+	"os/exec"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/dlvhdr/gh-dash/v4/ui/components/prssection"
+	"github.com/dlvhdr/gh-dash/v4/ui/constants"
+	"github.com/dlvhdr/gh-dash/v4/ui/context"
+)
+
+func (m *Model) approve(comment string) tea.Cmd {
+	pr := m.pr.Data
+	prNumber := pr.GetNumber()
+	taskId := fmt.Sprintf("pr_approve_%d", prNumber)
+	task := context.Task{
+		Id:           taskId,
+		StartText:    fmt.Sprintf("Approving pr #%d", prNumber),
+		FinishedText: fmt.Sprintf("pr #%d has been approved with comment: %s", prNumber, comment),
```

Comment:
Let's not include the content of the comment as it can be quite long and have new lines. This will break the UI once rendered

Reply:
Makes sense, adjusted. Thanks!